app/main.py

```
@@ -1,3 +1,89 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> Any:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km + other)
+        else:
+            raise TypeError(f"Unsupported operand type for + : {type(other)}!")
+
+    def __iadd__(self, other: Any) -> Any:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        elif isinstance(other, (int, float)):
+            self.km += other
+            return self
+        else:
+            raise TypeError(f"Unsupported operand type for + : {type(other)}!")
+
+    def __mul__(self, other: Any) -> Any:
+        if isinstance(other, (int, float)):
+            return Distance(km=self.km * other)
+        else:
+            raise TypeError(f"Unsupported operand type for * : {type(other)}!")
+
+    def __truediv__(self, other: Any) -> Any:
+        if isinstance(other, (int, float)) and other != 0:
+            return Distance(km=round((self.km / other), 2))
+        elif other == 0:
+            return print("Can't division by 0!")
+        else:
+            raise TypeError(f"Unsupported operand type for / : {type(other)}!")
+
+    def __lt__(self, other: Any) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            print(f"the {other} is not a Distance and it doesn't a number!")
+            return False
```

Issue: In the `__lt__` method, for unsupported operand types, you are printing a message and returning `False`. According to the checklist, you should raise a `TypeError` for unsupported types in operator overloads, not print a message or return `False`.